### PR TITLE
fixes breaking changes when updated to jekyll 3.0

### PR DESCRIPTION
**Pull request by seyfeddin**
Sorry for the last one. This PR should be fine.

Since GH Pages updated its Jekyll version to 3.0, documentation on ComponentKit's website was broken (scroll to the bottom of the page on any documentation detail page).

This PR fixes ordering by prepending ids to filenames, corrects links, adds dependency for pygments.rb (Jekyll 3.0 dropped its support for pygments and moved to Rouge, which does not highlight ObjC++ files).

### CODE REVIEW

File: _docs/01-getting-started.md [hunk 1]
```diff
@@ -67,5 +67,5 @@ Run the app once more. Now it looks a lot better:
 Congratulations! You've done your first development with ComponentKit. Keep poking around the sample app to learn more, or start using it in your own apps. Just add the following to your Podfile:
 
 ```
-pod 'ComponentKit', '~> 0.14'
+pod 'ComponentKit', '~> 0.12'
```
Comment (ocrickard) [inline _docs/01-getting-started.md]: Why this change?

**Comment by seyfeddin** [comment]
sorry, fixed now.

**Comment by ocrickard** [comment]
Ideally I think these commits would be squashed. What do you think?

**Comment by ocrickard** [comment]
Other than that though looks fine to me.

**Comment by ocrickard** [comment]
Alright, I'm going to go ahead and merge this. Thanks for fixing!

**Comment by ocrickard** [comment]
And would you look at that... the site's fixed!
